src/Products.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+namespace Netzstrategen\ShopStandards;
+
+/**
+ * Meant to handle special Product features.
```

Comment:
This is a very general description and does not create much value, I think it explain exactly why it is trying to accomplish, it could be something in the line of
```suggestion
 * Allows to optionally include the main category in the permalink for SEO.
```

Reply:
if we settle down our discussion about the class name itself we can also settle this one ;)